actions/xmpp

```diff
@@ -230,6 +234,67 @@ def subcommand_change_domainname(arguments):
     action_utils.service_start('ejabberd')
 
 
+def subcommand_add_letsencrypt(arguments):
+    """Start using letsencrypt certificate if it matches configured domain."""
+    with open(EJABBERD_CONFIG, 'r') as file_handle:
+        conf = ruamel.yaml.round_trip_load(file_handle, preserve_quotes=True)
+
+    if arguments.domain in conf['hosts']:
+        cert_folder = '/etc/ejabberd/letsencrypt/' + arguments.domain
+        if not os.path.exists(cert_folder):
+            os.makedirs(cert_folder)
+            shutil.chown(cert_folder, 'ejabberd', 'ejabberd')
+
+        cert_file = cert_folder + '/ejabberd.pem'
+        le_folder = '/etc/letsencrypt/live/' + arguments.domain
+        with open(cert_file, 'w') as outfile:
+            with open(le_folder + '/privkey.pem', 'r') as infile:
+                for line in infile:
+                    if line.strip():
+                        outfile.write(line)
+            with open(le_folder + '/fullchain.pem', 'r') as infile:
+                for line in infile:
+                    if line.strip():
+                        outfile.write(line)
```

Comment:
If we copy the certs from letsencrypt directory, then when letsencrypt renews its certificate every 3 months, those certificates won't get reflected for ejabberd. Many daemons such as Apache first run as root, read the certificates before demoting themselves into the role of a regular user.  Won't ejabberd work like this?